### ISSUE TITLE
ufunc coverage changed in numpy dev

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#6988]
Example log: https://travis-ci.org/astropy/astropy/jobs/316592597

```
_______________________ TestUfuncCoverage.test_coverage ________________________
self = <astropy.units.tests.test_quantity_ufuncs.TestUfuncCoverage object at 0x7f796e3cd470>
    def test_coverage(self):
        all_np_ufuncs = set([ufunc for ufunc in np.core.umath.__dict__.values()
                             if type(ufunc) == np.ufunc])

        from .. import quantity_helper as qh

        all_q_ufuncs = (qh.UNSUPPORTED_UFUNCS |
                        set(qh.UFUNC_HELPERS.keys()))

>       assert all_np_ufuncs - all_q_ufuncs == set([])
E       AssertionError: assert {<ufunc 'gcd'>, <ufunc 'lcm'>} == set()
E         Extra items in the left set:
E         <ufunc 'gcd'>
E         <ufunc 'lcm'>
E         Use -v to get the full diff
```

**Comment by bsipocz** [comment]
@mhvk - I suspect this needs to be addressed by 3.0?

**Comment by mhvk** [comment]
See #7026
